### ISSUE TITLE
Game Request: [Leaf] ToHeart2 AnotherDays

### ISSUE THREAD

**Issue by seven-nights** [vn-tools/arc_unpacker#56]
## Game info

VNDB link: https://vndb.org/v90
Developer: Leaf
Release date: 2008
Original title: ToHeart2 AnotherDays

## Details
Game request
I'm trying to unpack the voice files in this game. While I can unpack the .a files it uses, the files I get are in .g format which nothing seems to recognise.

**Comment by rr-** [comment]
I plan to support this at some point but for time being I'm tired of Leaf [rewriting their engine every game](https://github.com/vn-tools/arc_unpacker/tree/master/src/dec/leaf/). I'll take care of other games first.

**Comment by seven-nights** [comment]
Now that you mention it, they do like to experiment with their engine... Fair enough, I can wait for it.

**Comment by rr-** [comment]
I pushed a few commits that add enhance support for various Leaf formats, one of which decodes `.g` files.

`.px` files need to wait till I figure out how to differentiate max block size between 「Utawarerumono」 and and 「Tenshi no Inai 12-gatsu」.

**Comment by TheFishtare** [comment]
Sorry but, Are you still planning to release a new release of this Project?

**Comment by rr-** [comment]
Not really. If someone wants to take over the maintenance, they're free to contact me on my e-mail.

**Comment by TheFishtare** [comment]
Well, if so; "I'll be back"
